meta/system.py

```diff
@@ -46,6 +46,7 @@ def __init__(self):
         self.cart_questions = {}
         self.normalization_matrices = {}
         self.stm_files = {}
+        self.glm_files = {}
```

Comment:
There is a `create_stm_from_corpus`, is there no need for something similar for the glm?

Reply:
> Because now you would need to set this by explicitely accesing the variable...

we could create a generic setter function for the glm files if you prefere it.

Reply:
> we could create a generic setter function for the glm files if you prefere it.

I would suggest this or have a verbose error message if the glm file is not set yet

Reply:
But still we can merge this now

Reply:
> I would suggest this or have a verbose error message if the glm file is not set yet

I added that. If you have no objections to the message, we can merge the PR.